gruntfile.js

```diff
@@ -201,8 +203,16 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-contrib-stylus');
     grunt.loadNpmTasks('grunt-autoprefixer');
     grunt.loadNpmTasks('grunt-gh-pages');
+    
+    grunt.registerTask('grok', 'Fire up ngrok so icanhaz https.', function () {
```

Comment:
I was thinking the same thing. There is already a grunt plugin `grunt-ngrok`. However it forces you to use a chosen subdomain, which requires an ngrok account and auth token, and was written around that concept so a PR was not an option. So in the interest of preserving namespace I chose `grok`. I can change to `ngrok` though.